src/components/DataTable/DataTable.jsx

```diff
@@ -267,6 +274,11 @@ const DataTable = createClass({
 		);
 		const allSelected = _.every(data, 'isSelected');
 
+		const passActiveWidth = (childData, obj) => {
+			var activeWidth = Object.assign({}, this.state).activeWidth;
+			activeWidth[obj.props.children] = childData;
+			this.setState({ activeWidth });
```

Comment:
```suggestion
this.setState({ activeWidth: Object.assign({}, this.state.activeWidth, { [obj.props.children]: childData })});
```

maybe use a different name than `children` - it might be confused with React `children` prop